query.js

```diff
@@ -675,6 +675,7 @@ function parseExport(res) {
             endYear: a.travelEndYear || "",
             endMonth: a.travelEndMonth || "",
             endDay: a.travelEndDay || "",
+            markers: a.markers || "",
```

Comment:
@ryanctan what is the data type of `a.markers`? If it is an array (I think it is?), then we may need to convert it to some kind of string, as it will only show up in a single column.

Reply:
So, currently markers are exported like so:
![image](https://user-images.githubusercontent.com/51471027/60763381-3764ce00-a028-11e9-9cb8-b85caf8d71fb.png)
Is there a different way they should be exporting?

Reply:
oh I see, `markers` is a single value, not a list. Got it -- thanks!